doc/jupyter_lite_config.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Why is there a `jupyter-lite.json` and `jupyter_lite_config.json`? These seem redundantly (or at least non-descriptively) named.

Reply:
I've always been confused about this myself. One is for the CLI, one is for the JupyterLite frontend. The naming is pretty bad.

Reply:
The JupyterLite CLI is a `traitlets` based app, so `jupyter_lite_config.json` follows the naming convention as other apps, like `voila.json` or `jupyter_server_config.json`. It is indeed for build time configuration (when running the JupyterLite CLI, for example with `jupyter lite build`).

`jupyter-lite.json` is for runtime configuration when loading the page. It is similar to the `page_config.json` used in JupyterLab.

Agree this can be confusing. Maybe JupyterLite could also support loading `page_config.json` files so users could choose a different name for the file?

For more information: https://jupyterlite.readthedocs.io/en/stable/howto/configure/config_files.html